build_tools/azure/build_and_deploy.yml

```diff
@@ -7,7 +7,7 @@ steps:
     pip install -r build_tools/azure/requirements.txt
   displayName: 'Installing requirements'
 
-- script: python setup.py bdist_wheel
+- script: make bdist_wheel
```

Comment:
will this work on windows? is cmake installed?

Reply:
It works on [Azure Pipelines](https://dev.azure.com/tgsmith61591gh/pmdarima/_build/results?buildId=729). If anyone wants to develop locally on Windows, they would need cmake for it to work...

Reply:
https://github.com/Microsoft/azure-pipelines-image-generation/blob/master/images/win/Vs2017-Server2016-Readme.md#cmake

Reply:
Thanks for the sleuthing